Frontend/src/app/create-profile/create-profile.component.html

```diff
@@ -0,0 +1,41 @@
+<mat-toolbar id="createProfileHeader">
+    <button mat-icon-button class="example-icon" [matMenuTriggerFor]="menu" aria-label="Example icon-button with menu icon">
+      <mat-icon [ngStyle]="{'color':'orangered'}">menu</mat-icon>
+    </button>
+    <span>Create Profile</span>
+    <span class="example-spacer"></span>
+    <mat-menu #menu="matMenu">
+        <button mat-menu-item (click)="showHomePage()">
+          <mat-icon [ngStyle]="{'color':'orangered'}">home</mat-icon>
+          <span>Home</span>
+        </button>
+        <button mat-menu-item>
+          <mat-icon [ngStyle]="{'color':'orangered'}">info</mat-icon>
+          <span>About</span>
+        </button>
+      </mat-menu>
+  </mat-toolbar>
+
+  <div id="mainPageContainer">
+  <form [formGroup]="createProfileForm">
+    <div id="createUserFields">
+        <mat-form-field color="warn">
+            <mat-label for="firstName">First Name</mat-label>
+            <input matInput formControlName="firstName">
+        </mat-form-field>
+        <mat-form-field color="warn">
+            <mat-label for="lastName">Last Name</mat-label>
+            <input matInput formControlName="lastName">
+        </mat-form-field>
+        <mat-form-field color="warn">
+            <mat-label for="bio">Bio</mat-label>
+            <input matInput formControlName="bio">
+        </mat-form-field>
+        <mat-form-field color="warn">
```

Comment:
I am not familiar with these tags. Is this a component library?

Reply:
Yeah, essentially the "mat" stuff is part of the angular materials library. So, it's basically like a "pre-formatted" input tag. I found it from here: https://material.angular.io/components/input/overview

Reply:
Ok awesome!